install.sh

```diff
@@ -151,6 +151,16 @@ if [[ "${ARCH}" = "x86_64" ]] && [[ -f "/lib/ld-2.23.so" ]]; then
   PREFIX_CMD="linux32 env LD_LIBRARY_PATH=${CREW_PREFIX}/lib${CREW_LIB_SUFFIX}:/usr/lib${CREW_LIB_SUFFIX}:/lib${CREW_LIB_SUFFIX}"
 fi
 
+PATCHELF_INTERPRETER=/usr/local/bin/ld.so
```

Comment:
Abstract the path with `${CREW_PREFIX}`?

Reply:
Done.